app/config.default.coffee

```diff
@@ -47,6 +47,8 @@ module.exports =
   autoSuggest:
     sortOrder: locality: 1, address: 2, stop: 3
     sortOthers: 4
+  # Let Pelias suggest based on current user location
+  locationAwareSuggestions: false
```

Comment:
Our config is currently a bit messy with most things being directly on the root level. I would appreciate if you could move the option to be within autoSuggest in order to group options that are relevant together.